purchase_operating_unit/__manifest__.py

```diff
@@ -0,0 +1,27 @@
+# © 2015-17 Eficent Business and IT Consulting Services S.L.
+# - Jordi Ballester Alomar
+# © 2015-17 Serpent Consulting Services Pvt. Ltd. - Sudhir Arya
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+{
+    "name": "Operating Unit in Purchase Orders",
+    "summary": "An operating unit (OU) is an organizational entity part of a "
```

Comment:
Improve the summary: such as
```suggestion
    "summary": "Adds the concept of operating unit (OU) in purchase order management "
```

Reply:
Updated summary